exec/java-exec/src/main/java/org/apache/drill/exec/metastore/store/SimpleFileTableMetadataProvider.java

```diff
@@ -97,47 +100,38 @@ public FileMetadata getFileMetadata(Path location) {
 
   @Override
   public NonInterestingColumnsMetadata getNonInterestingColumnsMetadata() {
-    return null;
+    return new NonInterestingColumnsMetadata(Collections.emptyMap());
```

Comment:
Better name? `NonInteresting` --> `Uninteresting`?

But, why are they not interesting? Is it because they we don't track their statistics? Is so, maybe we can define a term "tracked? untracked?" to describe those columns which do (or do not) participate in statistics. I did a quick check or Oracle, Postgres and SqlServer, but I didn't find a good term. Maybe "excluded columns"?

Reply:
Yes, that's correct that we don't track stats for such columns. This naming came from parquet metadata cache improvements and left to be consistent with that code.